jobs/vault/templates/data/properties.sh.erb

```diff
@@ -10,10 +10,11 @@ export JOB_INDEX=<%= index %>
 export JOB_FULL="$NAME/$JOB_INDEX"
 
 <%
-vault_addr=p("vault.address", "")
+vault_addr=p("vault.addr", "127.0.0.1")
```

Comment:
Slightly different than the template default. If we want to make them the same this should be `https://127.0.0.1:8200`. Alternatively, we could not default them here and depend on the spec to do it for us.